### ISSUE TITLE
Simplify the interface - count of elements and attributes

### ISSUE THREAD

**Issue by prantlf** [rhysd/bytesize-icon-elements#9]
I opened several pull requests for improving the interface of your library. Each contains a single commit and they are supposed to be applied from the oldest one to the newest one. They are based on each other, because most of them depend on changes done in the previous one. (I couldn't base all of them on your master; I would've had to be resolving conflicts a lot, I'm sorry.)

Except for some chore changes and a new attribute, eventually I modified the interface from separate elements like this:

```html
<icon-alert width="32" height="32" type="thin"></icon-alert>
```

to the single element with the `name` attribute:

```html
<bytesize-icon name="alert" size="32" weight="thin" style="miter"></bytesize-icon>
```

Will you accept this change in your library, or do you think that it should belong to a different library? Thank you!

**Comment by rhysd** [comment]
Thank you for the many PRs. I merged them and published a new version v2.0.0.

Actually I'm not developing this package actively. So, if you want, I'd like to add you as a committer. (Either creating a new org `bytesize-icon-elements` or adding you to repo as collaborator is OK.)

What do you think?

**Comment by rhysd** [comment]
Anyway this issue was resolved. Closing.
